src/main/java/frc/robot/commands/ElevatorDefaultCommand.java

```diff
@@ -33,31 +36,39 @@ public void initialize() {
 
     @Override    
     public void execute(){
-        ChassisSpeeds chassisSpeeds = s_Swerve.getEstimatedFieldRelativeSpeeds();
-        
-        SmartDashboard.putNumber("xSpeed", chassisSpeeds.vxMetersPerSecond);
-        SmartDashboard.putNumber("ySpeed", chassisSpeeds.vyMetersPerSecond);
+    
+        Pose2d botPose = s_Swerve.getLimelightBotPose();
 
-        if (DriverStation.getAlliance().get().equals(Alliance.Red)) {
-            speakerCoordinate = new Pair<Double, Double>(8.0, 1.5);
-        } else {
-            speakerCoordinate = new Pair<Double, Double>(-8.0, 1.5);
-        }
+        boolean isRed = DriverStation.getAlliance().get().equals(Alliance.Red) ? true : false;
 
-        Pose2d botPose = elevatorSubsystem.s_Limelight.getBotPose(); 
-        botPose = s_Swerve.getLimelightBotPose();
+        SmartDashboard.putBoolean("Is Red Alliance", isRed);
 
-        double xDiff = botPose.getX() - speakerCoordinate.getFirst(); // gets distance of x between robot and target
-        double yDiff = botPose.getY() - speakerCoordinate.getSecond();
+        double xDiff = botPose.getX() - (isRed? 8.3:-8.3);
+        double yDIff = botPose.getY() - 1.45;
+        double hypotenuse = Math.sqrt(Math.pow(xDiff, 2) + Math.pow(yDIff, 2));
 
-        double distance = Math.sqrt(yDiff * yDiff + xDiff * xDiff);
+        double feetDistance = Units.metersToFeet(hypotenuse);
 
-        double target = -.00384 * distance * distance + 1.17 * distance - 94.8; //NEEDS TO BE REPLACED LATER WITH THE TARGET ANGLE
-    
-        double elevatorValue = elevatorSubsystem.getElevatorValue();    
-        angleShooterUtil.updateTargetDiff(elevatorValue - target);
+        SmartDashboard.putNumber("Hypotenuse", feetDistance);
+
+        double angle = Units.radiansToDegrees(Math.atan2(Constants.speakerHeightMeters, hypotenuse));
+
+        SmartDashboard.putNumber("ElevatorDegrees", angle);
+
+        // 29.6 min     59.9 max
 
-        SmartDashboard.putNumber("ShooterEncoder", elevatorValue);
+        double degreesToEncoderAngle = (angle - 29.6) * -2.03630363036303630363;
```

Comment:
We should calculate -2.03630363036303630363 in constants based on the min and max angle and max encoder value.